lib/core/userinfo/components/user-info.component.ts

```diff
@@ -70,7 +70,7 @@ export class UserInfoComponent implements OnInit, OnDestroy {
     selectedIndex: number;
     private destroy$ = new Subject();
 
-    constructor(private ecmUserService: EcmUserService,
+    constructor(private ecmUserService: PeopleContentService,
```

Comment:
I suggest we name the service to match the service class name, in this case `peopleContentService`